app/assets/stylesheets/webapp/ncr.scss

```diff
@@ -7,3 +7,13 @@
     margin-bottom: 20px;
   }
 }
+
+.controller-gsa18f-proposals {
+  .content.container {
```

Comment:
Do we think these two (`controller-ncr-proposals` and `controller-gsa18f-proposals`) are going to diverge much? If not, let's just use a single `proposal-form` or similar class